content/support/_index.md

```diff
@@ -0,0 +1,33 @@
+---
+title: "Support"
+---
+
+# I need help with PrestaShop
+
+The PrestaShop open-source project can be freely downloaded and used by anyone, but it is provided "as is", with no warranty or dedicated support. 
+
+Rest assured, there are many resources to help you if you need assistance.
+
+## Community help
+
+PrestaShop has a sizeable community, and there are many volunteers willing to help each other on the web:
+
+* [Report a bug on GitHub]({{< devdocs "contribute/contribute-reporting-issues/" >}})
+* [Join the community forums](https://www.prestashop.com/forums/)
+* [Ask on Stack Overflow](https://stackoverflow.com/questions/tagged/prestashop)
+* [Discuss on Slack]({{< relref "/slack" >}})
+
+## Contact a professional
+
+If you need serious help with your project, your best choice is to get contact a professional. There are thousands of agencies and freelance experts located around the globe:
```

Comment:
"your best choice is to get contact a professional"

I think maybe there's a little typo here?

-> your best choice is to get in touch with a professional
-> your best choice is to contact a professional

Reply:
Good catch, thanks
```suggestion
If you need serious help with your project, your best choice is to contact a professional. There are thousands of agencies and freelance experts located around the globe:
```